builder/builder.go

```diff
@@ -153,6 +157,17 @@ func (b *Builder) Build(ctx context.Context, payload *Payload) (*monomer.Block,
 		return nil, fmt.Errorf("commit: %v", err)
 	}
 
+	ethState, err := state.New(b.blockStore.HeadBlock().Header.StateRoot, b.ethstatedb, nil)
+	if err != nil {
+		return nil, fmt.Errorf("create ethereum state: %v", err)
+	}
+	// TODO: execute withdrawals
+	ethStateRoot, err := ethState.Commit(uint64(currentHeight+1), true)
```

Comment:
to avoid counting twice
```suggestion
	ethStateRoot, err := ethState.Commit(uint64(header.Height), true)
```